8.2.18 Release # DO NOT DELETE

su-sws-jira opened the pull request.

su-sws-jira committed:
8.2.18

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

pookmish committed:
Merge branch 'develop' into 8.2.18-release

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

kaseybon committed:
Sparkbox Sprint 24 (#711)

* fix(STN-524): adjust spacing for lead and splash fonts inside a list (#701)

* feat(STN-506): carousel height refinement (#703)

JIRA STN-506 https://sparkbox.atlassian.net/browse/STN-506

* feat(STN-504): Increase Traditional Font Size (#707)

* feat(STN-504): increase the font-size and line-height for the traditional theme

- increase heading in ptype-hs-row to 22px in the traditional theme
- set field-content font-size to 18px in traditional theme
- decrease 3rd column font size in the traditional theme
- increase hs-caption and hs-credits font size in traditional theme
- create hb-credits mixin and use for figcaption
- increase card title font-size on mobile for traditional theme
- decrease body line-height, increase highlighted card text size

* chore: compiled css

* feat(STN-523): fix spacing on vertical linked cards on mobile (#705)

JIRA STN-523 https://sparkbox.atlassian.net/browse/STN-523

* fix(STN-505): Update utility classes to work with Drupal Views (#704)

* feat(STN-505): update utility classes to work in views

* update: added reset to more styles
* update: added to hb views classes
* update: hb-highlighted-label
* update: wysiwyg spacing
* update: hb-title, hb-blockquote, hb-highlighted-label for views
* update: refactored views styling and added comments

* feat(STN-472): Hero Height Refinement (#709)

* feat (STN-472): hero height refinement

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

* feat(STN-348): Add Font Family Selection for Traditional Theme (#710)

* feat: added font family selection for traditional

- Added Font Family Selection in Traditional theme settings, serif is default.

* styles: refactor SCSS comments and spacing

* chore: compiled css

* fix: resolve codeclimate errors

Co-authored-by: Heather Taylor <heather@heysparkbox.com>
Co-authored-by: KelbelR <kelbel@heysparkbox.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

joegl committed:
Provisioned 5 sites (#714)

Co-authored-by: joegl <joegl@users.noreply.github.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

pookmish committed:
HSD8-851 Convert event date to use smart date field type (#698)

* HSD8-851 Switch to using the smart date module

* 8.2.17

* Updated dependencies

* Built out the update hook to move event date stuff around

* added module enable to update hook

* fixed module install

* delete field storage if it exists

* Default configs and content updates (#699)

* updated default configs and content

* updated test

* Adjusted test

* Updated dependencies Aug 26 2020

* Sparkbox sprint 23 (#691)

* feat(STN-373): added lando and CSS documentation for developers (#671)

* feat(STN-481): update the default color pairing for Colorful to Cardinal (#685)

* fix(STN-492): secondary nav header spacing and border-top (#684)

* fix(STN-492): secondary nav header spacing and border-top

* fix(STN-492): slight margin adjustment to Colorful

* chore(STN-492): compiled CSS

* STN-480:  Fix spacing beside navigation dropdown toggle (#686)

* fix (STN-480): add spacing to navigation dropdown arrow in nested toggler

- When all pages within a nested toggler are private, the class of hb-main-nav__item--parent is not added. Without this class the icon beside nav items doesn't recieve appropriate spacing. Resolve by adding spacing to the nested toggler.

* feat: add compiled CSS

* STN-495: Add the Warbler Color Pairing (#688)

* feat(STN-495): warbler color pairing and local dark footer changes for tradl

* feat(STN-495): warbler theme settings

* chore(STN-495): compiled css

* feat (STN-490): refactor hb-grid system

JIRA STN-490 https://sparkbox.atlassian.net/browse/STN-490

* feat: refactor paragraph type grid system to use CSS Grid

* Feat 392  behat (#680)

* wip: starting on behat test for row style

* update: row paragraph test add spotlight test

Co-authored-by: Lindsay Silver <lindsay@heysparkbox.com>

* feat (STN-493): add title region to 3 column w/image layout

JIRA STN-493 https://sparkbox.atlassian.net/browse/STN-493

* feat (STN-522): create utility class to display title text on Colorful arrow link

JIRA STN-522 https://sparkbox.atlassian.net/browse/STN-522

* STN-520: spacing bug fix for hb-views-divider class (#697)

* fix(STN-520): adjust views divider hb-card spacing from qa feedback

I added padding to the hb-views-class, and removed the margin.  Padding will apply within the container above the divider being added and not outside of the container as margin does.

* chore: compiled css

* chore: compiled CSS

* fix: Include the new title region in the 3 col with image layout module

Co-authored-by: Heather Taylor <heather@heysparkbox.com>
Co-authored-by: Lindsay R. Silver <lindsay@heysparkbox.com>
Co-authored-by: KelbelR <kelbel@heysparkbox.com>

* adjusted test

* Flush caches

* Added cancel button to entity forms

* update hook for event views

* Update event display configs

* cc fixes

* cc fixes

* added missing third party settings

* updated display modes

* HSD8-903 Switch to use the contrib module for revision deleting

* HSD8-894 Adjust some settings

* uninstall revision cleanup module;

* removed timezone compenstation

* truncate seconds from event date and set the form display for the field

* save edited form display

Co-authored-by: CircleCI <sws-developers@lists.stanford.edu>
Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>
Co-authored-by: Heather Taylor <heather@heysparkbox.com>
Co-authored-by: Lindsay R. Silver <lindsay@heysparkbox.com>
Co-authored-by: KelbelR <kelbel@heysparkbox.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

pookmish committed:
updated dependencies

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

pookmish committed:
Fixed behat steps (#718)

* changed behat dependency version

* try older compserlock

* try different standalone chrome

* patch drupal-extension module

* adjusted composer

* trim email on import

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

kaseybon committed:
STN-553: Credits and Captions (#716)

* refactor(STN-553): unnest the hs- utility classes

* fix(STN-553): hs-caption and hs-credits were getting the wrong font-size on Traditional

- Removed duplicate hs-caption style with outdated code
- Move the reset mixin to the hb-caption mixin
- Make sure figcaptions get the italic styles
- Reorder mixins so that the reset mixin is at the top of the file

* chore: compiled CSS

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs

Commit:
STN-556 : Rollback Carousel and Hero Refinements (#719)

* feat: rollback carousel height refinements

- reincorporate the carousel controls alignment fix

* feat: rollback hero height refinements

* feat: re-add compiled CSS

pookmish committed:
HSD8-906 HSD8-905 Provision clas and impact sites

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr708-cxbsn9ghzeqcizrggh5cfdv1b6xo8ejo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0

Visual Diffs:
- https://dashboard.tugboat.qa/5f4ee57923b4003a30231ba0/visualdiffs
